ANTLR/BaseVisitor.py

```diff
@@ -132,6 +137,15 @@ def visitRemove(self, ctx: GoPoParser.RemoveContext):
         self.tmp_memory['list'].remove(number_to_remove)
         return self.visitChildren(ctx)
 
+    def visitFilter(self, ctx:GoPoParser.FilterContext):
+        operator = self.visit(ctx.getChild(2))
```

Comment:
filter is defined as
`FILTER OPAR op=(EQ | NEQ | GT | LT | GTEQ | LTEQ) NUMBER CPAR`

Probably better way will be:

```
operator = ctx.op.type
function = lambda_two_args_by_operator[operator]
```

After this change, there will be no use for func `get_function_from_operator`
What's more, we will use specific TOKENS and not operate with texts which is a proper approach in my opinion

Reply:
Yes, you're right - I changed it.